tensorbay/opendataset/CACD/catalog.json

```diff
@@ -0,0 +1,6040 @@
+{
+    "CLASSIFICATION": {
+        "categories": [
```

Comment:
Try to format this json file to reduce the number of the total lines.
reference: https://github.com/linjiX/tensorbay-python-sdk/blob/f0704a9f0d020d1b2b4f394adb367269b90f34e5/tensorbay/opendataset/WIDER_FACE/catalog.json